kore/src/Kore/AST/Common.hs

```diff
@@ -492,6 +558,14 @@ instance Unparse child => Unparse (Ceil level child) where
         <> parameters [ceilOperandSort, ceilResultSort]
         <> arguments [ceilChild]
 
+    unparse2 Ceil { ceilChild } =
```

Comment:
The implementation of this should be
```.hs
Pretty.parens (Pretty.fillSep ["\\ceil", unparse2 ceilChild])
```
to preserve spacing and indentation. The same is true for the other patterns.